.circleci/config.yml

```diff
@@ -12,10 +12,10 @@ parameters:
 
 references:
   envoy-versions: &supported_envoy_versions
-    - &default_envoy_version "1.22.7"
```

Comment:
Talked w/ @jmurret about [this backport](https://github.com/hashicorp/consul/commit/663641b508c063a11d6f26032c62d055805f21a2) which incorrectly updated these versions on the 1.13.x release branch. Temporarily restoring them to the correct versions here, but it won't matter for long since a PR later this week will remove this whole workflow